chat-plugins/trivia.js

```diff
@@ -1005,8 +1005,10 @@ const commands = {
 	submit: 'add',
 	add: function (target, room, user, connection, cmd) {
 		if (room.id !== 'questionworkshop') return this.errorReply('This command can only be used in Question Workshop.');
-		if (cmd === 'add' && !this.can('mute', null, room) || !target) return false;
-		if (!this.canTalk()) return;
+		if ((cmd === 'add' && !this.can('mute', null, room)) ||
+            (cmd === 'submit' && !this.can('broadcast', null, room)) ||
+            !target) return false;
+		if (!this.canTalk()) return false;
```

Comment:
``if (!this.canTalk()) return;``

Reply:
Returning undefined vs. false doesn't make a difference here as far as I know. The style I use for the trivia plugin errs towards returning false rather than undefined in most cases